include/swift/AST/DiagnosticsFrontend.def

```diff
@@ -136,6 +136,9 @@ ERROR(error_underlying_module_not_found,none,
 ERROR(error_unable_to_load_supplementary_output_file_map, none,
       "unable to load supplementary output file map '%0': %1", (StringRef, StringRef))
 
+ERROR(error_missing_entry_in_supplementary_output_file_map, none,
+      "the supplementary output file map '%0' is missing an entry for '%1' (this likely indicates a compiler issue; please file a bug report)", (StringRef, StringRef))
```

Comment:
Nitpick: no "the". (You can see a writeup I did about diagnostic phrasing in [docs/Diagnostics.md](https://github.com/apple/swift/blob/master/docs/Diagnostics.md).)

Reply:
Fixed.